tasks/coverage/parser_babel.snap

```diff
@@ -187,6 +183,22 @@ Expect to Parse: "typescript/class/constructor-with-modifier-names/input.ts"
  4 │ }
    ╰────
 
+Expect to Parse: "typescript/class/parameter-properties/input.ts"
```

Comment:
In this case, report errors are expected. I tested it in typescript playground the result was correct.